src/views/bootablevolumes/utils/types.ts

```diff
@@ -2,3 +2,7 @@ export type BootableVolumeMetadata = {
   labels: { [key: string]: string };
   annotations: { [key: string]: string };
 };
+
+export type InstanceTypesAndSizesMapType = {
```

Comment:
I would not use "To" substring in the naming because IMO it is misleading. We don't convert/change anything to size, size is part of the string/whole information we are dealing with. We just divide the whole label's string in here. 

Another note from me is that I don't think it's always good idea to follow some conventions/general recommendations. IMO readability of the code for a programmer as a higher priority to me, and in this case the suggested naming is less clear or self explaining than the original one.

Reply:
From what I can tell here, you're taking InstanceType codes such as `c4.xlarge` and breaking them down into the InstanceType name shorthand (c4) and the size (xlarge). You're then creating a mapping, like below. Correct me if I'm wrong. Perhaps I'm simply misunderstanding what the code is doing here.

```
const map = {
  c4: [xlarge,...],
  ...
}
``` 

If the above example is correct, "To" is a much more accurate term than "and" for this use case. A map connects the values to the keys by mapping the key TO the value. Using "and" implies a group of InstanceTypes and sizes together in one group/array, which is incorrect. Once you access the map using an InstanceType shorthand name, you only get the sizes for that InstanceType category, not a list of sizes AND InstanceType names.

I completely agree that readability is paramount. Conventions typically exist to improve readability not hinder it. I don't see how "XWithY" is clearer than "XtoYMap"/"XToY" or "YByX". I have no idea what "with" is supposed to mean in this instance.

In any case, "Type" shouldn't be used in the name.

Reply:
> In any case, "Type" shouldn't be used in the name.

I mostly agree, but not 100%. IMO, it depends on a situation...

Anyway, we don't have much time, so can you please check the actual version of the code? I think you are still looking at some old version of the code. It was updated :)

Reply:
Weird. I refreshed last night before commenting.